Source/Scene/Scene.js

```diff
@@ -3043,6 +3048,16 @@ define([
      * @exception {DeveloperError} Picking from the depth buffer is not supported. Check pickPositionSupported.
      */
     Scene.prototype.pickPosition = function(windowPosition, result) {
+        var cacheKey;
```

Comment:
As mentioned before, the caching belongs in `pickPositionWorldCoordinates`. For example, both `Camera` and `ScreenSpaceCameraController` call it instead of `pickPosition`.